modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -661,7 +663,7 @@ protected function register_controls() {
 			'global' => [
 				'default' => Global_Typography::TYPOGRAPHY_ACCENT,
 			],
-			'selector' => '{{WRAPPER}} .elementor-tab-title',
+			'selector' => '{{WRAPPER}} .e-tab-title-text',
 		] );
```

Comment:
I noticed that the typography (underline) was affecting the icon. I have targetted it more specifically now.